src/main/proto/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this file really necessary, it's not a proto file?

Reply:
Yes, it was added by libra but is not necessary for generation.